.github/workflows/full_suite_integration_tests.yml

```diff
@@ -161,6 +161,7 @@ jobs:
 
   trigger-pointer:
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```

Comment:
why this kind of check?

Reply:
Noticed during my work on this PR. Pointers should be only triggered by the master branch. We don't want comment trigger to change date.
If you want I can makanother PR for this change

Reply:
```suggestion
    if: ${{ github.event_name != 'issue_comment'}}
```

Reply:
Hm...same results, but sure, I can change it that way

Reply:
same, but we do not depend on the branch name.